frontend/src/AlertsManagement/AlertsContent.test.js

```diff
@@ -39,22 +39,44 @@ describe("handleTabs", () => {
 });
 
 describe("handleCheckbox", () => {
-  const editUnchecked = [1, 2, 3];
-  const editChecked = [4, 5, 6, 0];
-  const missingChecked = [4, 5, 6];
-  const doubleChecked = [0, 4, 5, 6];
-
-  const styles = { 
-    root: {
-      flexGrow: 1,
-      maxWidth: 900,
-      margin: 'auto',
-    }
-  };
+  // Mock data for the alerts, unresolved and resolved.
+  const fakeAlerts = new Map();
```

Comment:
```
map_name = new Map([
  [key1, {value1}],
  [key2, {value2}],
  [key3, {value3}],
])

unless you think its less clean that way?
```

Reply:
I followed the writing here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
"The correct usage for storing data in the Map is through the set(key, value) method."